samples/wifi/sr_coex/README.rst

```diff
@@ -157,15 +142,18 @@ To enable different test modes, set up the following configuration parameters in
 * Bluetooth LE configuration: Set the Bluetooth LE connection interval limits using the :ref:`CONFIG_INTERVAL_MIN <CONFIG_INTERVAL_MIN>` and :ref:`CONFIG_INTERVAL_MAX <CONFIG_INTERVAL_MAX>` Kconfig options.
   The units are 1.25 milliseconds.
   For example, ``CONFIG_INTERVAL_MIN=80`` corresponds to an interval of 100 ms (80 x 1.25).
-* Wi-Fi connection: Set the following options appropriately as per the credentials of the access point used for this testing:
+* Wi-Fi connection: Set the following Wi-Fi credentials in the :file:`prj.conf` file:
 
-  * :ref:`CONFIG_STA_SAMPLE_SSID <CONFIG_STA_SAMPLE_SSID>`
-  * :ref:`CONFIG_STA_SAMPLE_PASSWORD <CONFIG_STA_SAMPLE_PASSWORD>`
-  * :ref:`CONFIG_STA_KEY_MGMT_* <CONFIG_STA_KEY_MGMT_*>`
-  * :kconfig:option:`CONFIG_NET_CONFIG_PEER_IPV4_ADDR`
+* :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC` - This option enables static Wi-Fi configuration.
```

Comment:
Why can't we just refer to the `wifi_credentials` doc here? Checkout `options-from-kconfig` Sphinx extension in NCS.

Reply:
``wifi_credentials_options.txt`` is being included in all other samples to list ``Wi-Fi credentials``. Doing the same here.

Reply:
Sure, but that itself won't be enough

1. Add a heading for `configuration options`
2. If `Kconfig` then use `options-from-kconfig` for that sample
3. Then add Wi-Fi credentials: `.. include:: /includes/wifi_credentials_options.txt`